src/components/AudioNative/AudioNative.js

```diff
@@ -3,10 +3,11 @@ import PropTypes from 'prop-types'
 import { Howler } from 'howler'
 
```

Comment:
I think @sahil97 updated this too-- just tagging so you two can coordinate merge conflicts